iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -95,15 +91,15 @@
   </ItemGroup>
 
   <!-- NetStandard 2.1, NetStandard 2.0 and net472 -->
-  <ItemGroup Condition="'$(SupportsNetStandard20AndAbove)' == 'true'">
+  <ItemGroup Condition=" '$(TargetFramework)' != 'net451' ">
```

Comment:
The space between the double-quote and single quote is intentional?

Reply:
Interestingly, yes. It was the format followed in the csproj everywhere, so I kept it consistent. I am sure there is no scientific reason for it, I just wanted to have it be consistent with the pre-existing code.

Reply:
I think it makes it readable